src/main/resources/application.properties

```diff
@@ -125,6 +125,8 @@ quarkus.hibernate-search-orm.elasticsearch.max-connections=90
 # Allow localhost in particular
 %dev,staging.quarkus.http.cors.origins=/.*/
 %dev,staging.quarkus.http.header."Access-Control-Allow-Private-Network".value=true
+# disable indexing and fetching of html quarkiverse guides in tests/dev
+%dev,test.quarkiverseio.enabled=false
```

Comment:
I assume it takes forever, then? :grin:

Reply:
😃 yeah, depending on the network... usually for me, it's 2-3 minutes, but doing so for each test index... So I was thinking that we test it in a specific test where we need it, and leave all other tests without this extra download-parse step